.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,43 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+--->
+
+Fixed issues: <!--insert issue reference in this form: #XX #XX-->
+
+## Type of change
+
+<!--- Please delete options that are not relevant for your change --->
```

Comment:
If it's a checkbox list, why should I delete options too? Either I should check a box or delete.

Reply:
Correct, that may not be the right format for this section.

I am thinking about an action that checks if all tasks are complete (e.g: tasks 13/13). The correct format would be a radio button to permit only 1 option at a time, but I don't think it is supported.

Reply:
removing checkbox list and keep the "please delete options ..." notice.